src/main/java/hudson/plugins/gradle/GradleConsoleAnnotator.java

```diff
@@ -33,15 +37,19 @@ protected void eol(byte[] b, int len) throws IOException {
         // trim off CR/LF from the end
         line = trimEOL(line);
 
-        if (line.startsWith(":") || line.startsWith("> Task :"))
-            // put the annotation
-            new GradleTaskNote().encodeTo(out);
+        if (usesGradleBuilder) {
```

Comment:
I think we should add the console decoration if when not using the Gradle builder. Any reason this should not be done?

Reply:
Mainly because it seemed likely that arbitrary builds might contain lines prefixed with `:` that aren't necessarily Gradle tasks. It seemed a common enough token where folks might start to get funky highlighting in existing builds.

Reply:
Fine with me. The plugin should probably start using an init script for getting the information as Teamcity does. That probably won't help for arbitrary build step logs I guess...